L1Trigger/L1TCommon/python/customsPostLS1.py

```diff
@@ -149,16 +149,7 @@ def customiseSimL1EmulatorForPostLS1_25ns(process):
 def customiseSimL1EmulatorForPostLS1_Additional_HI(process):
     # set the Stage 1 heavy ions-specific parameters
     # all of these should eventually end up in a GT
-    if hasattr(process,'RCTConfigProducers'):
-        process.RCTConfigProducers.eicIsolationThreshold = cms.uint32(7)
-        process.RCTConfigProducers.hOeCut = cms.double(999)
-        process.RCTConfigProducers.eMinForHoECut = cms.double(999)
-        process.RCTConfigProducers.eMaxForHoECut = cms.double(999)
-        process.RCTConfigProducers.hMinForHoECut = cms.double(999)
-        process.RCTConfigProducers.eMinForFGCut = cms.double(999)
-    if hasattr(process,'caloStage1Params'):     
-        process.caloStage1Params.jetSeedThreshold = cms.double(0.)
-        process.caloStage1Params.regionPUSType = cms.string("zeroWall")
+    process.load('L1Trigger.L1TCalorimeter.caloStage1Params_HI_cfi')
```

Comment:
To my knowledge this settings are not used during RECO (we don't re-emulate L1 except during DQM and MC generation).